### DISCUSSION TITLE
应用程序无法正常启动

### DISCUSSION THREAD

**Discussion by TensorABC** [shuax/MouseInc.Settings#368] [Q&A]
![Snipaste_2022-09-04_20-00-20](https://user-images.githubusercontent.com/95612401/188312322-e8ff5c9a-5888-412e-91e7-06990163cfc1.png)
在第一次打开后离线界面无法连接，打开设置无法显示界面，两秒后程序闪退。多次后程序无法打开，有时会生成名为MouseInc.exe.dat的文件
应用程序无法正常启动(0xc0000005)。请单击"确定"关闭应用程序。
未开启杀毒软件，windows defender状态已卸载

**Comment by TensorABC** [comment]
有人看到吗？
我打算重装系统了，因为很多应用都在我的win11上出现了这种报错。

**Comment by shuax** [comment]
很多程序同样问题那就是系统问题了。

**Reply by TensorABC** [reply]
我在全新的win11在安装了英特尔® 驱动程序和支持助理


然后出现了一样的提示，卸载后重启删除重新复制一个mouseinc才能使用，不知道你能不能复现。

**Reply by TensorABC** [reply]
我靠我又打不开了

**Reply by TensorABC** [reply]
我没安装英特尔® 驱动程序和支持助理

的时候是完全可用的

**Reply by TensorABC** [reply]
啊我服了，我刚改的win10，然后把音效替换成win11的重启之后又tm的寄了

**Comment by TensorABC** [comment]
[Crash.zip](https://github.com/shuax/MouseInc.Settings/files/9608620/Crash.zip)
崩溃报错

**Comment by shuax** [comment]
看不出问题来

**Reply by TensorABC** [reply]
我也觉得，我刚搞个了卡巴斯基，然后去卡饭随便下了个hips，竟然好用了😂
